test_scripts/API/PerformAudioPassThru/004_ATF_P_PerformAudioPassThru_SUCCESS_Only_Mandatory_Params_with_audioPassThruIcon.lua

```diff
@@ -0,0 +1,109 @@
+-- as well as AudioPassThruIcon
+-- 1.3. AudioPassThruIcon exists at apps sub-directory of AppStorageFolder (value from ini file)
+-- 2. Performed steps
+-- Send PerformAudioPassThru (audioPassThruIcon, mandatory params) from mobile to SDL and check:
+--
+-- Expected result:
+-- SDL sends UI.PerformAudioPassThru (audioPassThruIcon, other params) to HMI
+-- SDL sends TTS.Speak to HMI
+---------------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonPostconditions = require('user_modules/shared_testcases/commonPreconditions')
+local testCasesForPerformAudioPassThru = require('user_modules/shared_testcases/testCasesForPerformAudioPassThru')
+local testCasesForPolicyTable = require('user_modules/shared_testcases/testCasesForPolicyTable')
+
+--[[ General Precondition before ATF start ]]
+commonSteps:DeleteLogsFiles()
+config.defaultProtocolVersion = 2
+
+testCasesForPolicyTable:precondition_updatePolicy_AllowFunctionInHmiLeves({"BACKGROUND", "FULL", "LIMITED"},"PerformAudioPassThru")
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('cardinalities')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+
+commonSteps:PutFile("Precondition_PutFile_With_Icon","icon.png")
+
+function Test:Precondition_Check_audioPassThruIcon_Existence()
+  testCasesForPerformAudioPassThru:Check_audioPassThruIcon_Existence(self)
+end
+
+function Test:Precondition_ActivateApp()
+  testCasesForPerformAudioPassThru:ActivateAppDiffPolicyFlag(self, config.application1.registerAppInterfaceParams.appName, config.deviceMAC)
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+
+function Test:TestStep_PerformAudioPassThru_MandatoryParameters_audioPassThruIcon_SUCCESS()
+  local CorIdPerfAudioPassThruOnlyMandatoryVD= self.mobileSession:SendRPC("PerformAudioPassThru",
+    {
+      samplingRate = "22KHZ",
+      maxDuration = 500500,
+      bitsPerSample = "16_BIT",
+      audioType = "PCM",
+      audioPassThruIcon =
+      { 
+        value = "icon.png",
+        imageType = "STATIC"
+      }
+    })
+
+  EXPECT_HMICALL("UI.PerformAudioPassThru",
+    {
+      appID = self.applications[config.application1.registerAppInterfaceParams.appName],
+      maxDuration = 500500,
+      muteAudio = true,
+      audioPassThruIcon = 
+      { 
+        value = "icon.png",
+        imageType = "STATIC"      
+  	  }
+    })
+  :Do(function(_,data)
+      self.hmiConnection:SendResponse(data.id, "UI.PerformAudioPassThru", "SUCCESS", {})
+    end)
+
```

Comment:
@maria-mih, please be sure that TTS is not sent
EXPECT_HMICALL("TTS.Speak"):Times(0)